.travis.yml

```diff
@@ -57,10 +58,28 @@ script:
   # Get AWS credentials when not a pull request build.
   - if [ "$TRAVIS_PULL_REQUEST" = "false" ]; then mkdir credentials; fi
   - if [ "$TRAVIS_PULL_REQUEST" = "false" ]; then wget https://www.amazontrust.com/repository/AmazonRootCA1.pem -O credentials/AmazonRootCA1.pem; fi
-  - if [ "$TRAVIS_PULL_REQUEST" = "false" ]; then echo -e $AWS_IOT_CLIENT_CERT > credentials/clientCert.pem; fi
-  - if [ "$TRAVIS_PULL_REQUEST" = "false" ]; then echo -e $AWS_IOT_PRIVATE_KEY > credentials/privateKey.pem; fi
+  - if [ "$TRAVIS_PULL_REQUEST" = "false" ]; then
+      if [ "$RUN_TEST" == "provisioning" ]; then
+        echo -e $AWS_IOT_GAMMA_CLIENT_CERT > credentials/clientCert.pem;
+      else
+        echo -e $AWS_IOT_CLIENT_CERT > credentials/clientCert.pem;
+      fi
+    fi
+  - if [ "$TRAVIS_PULL_REQUEST" = "false" ]; then 
+      if [ "$RUN_TEST" == "provisioning" ]; then
+        echo -e $AWS_IOT_GAMMA_PRIVATE_KEY > credentials/privateKey.pem;
+      else
+        echo -e $AWS_IOT_PRIVATE_KEY > credentials/privateKey.pem;
+      fi
+    fi
```

Comment:
can this 2 ifs combined?

Reply:
The nested ifs are only temporary for running Provisioning tests/demos against the Gamma endpoint. I will create a separate PR to remove these

Reply:
Ok